app/data/jsonSchemaForm/emissionIntensityUiSchema.ts

```diff
@@ -0,0 +1,130 @@
+export const emissionIntensityReportingRequirementUiSchema = {
+  "ui:order": [
+    "reportDueDate",
+    "submittedDate",
+    "comments",
+    "measurementPeriodStartDate", // aka TEIMP Start Date
+    "measurementPeriodEndDate", // aka TEIMP End Date
+    "emissionFunctionalUnit",
+    "productionFunctionalUnit",
+    "baselineEmissionIntensity",
+    "targetEmissionIntensity",
+    "postProjectEmissionIntensity",
+    "totalLifetimeEmissionReduction",
+    "calculatedEiPerformance", // aka GHG Emission Intensity Performance
+    "adjustedEmissionsIntensityPerformance", // aka GHG Emission Intensity Performance (Adjusted)
+    "dateSentToCsnr",
+    "paymentPercentage", // aka Payment Percentage of Performance Milestone Amount (%)
+    "actualPerformanceMilestoneAmount", // aka Actual Performance Milestone Amount
+    "holdbackAmountToDate", // aka Maximum Performance Milestone Amount
+  ],
+  emissionFunctionalUnit: {
+    "ui:widget": "TextWidget",
+    classNames: "functional-unit",
+  },
+  productionFunctionalUnit: {
+    "ui:widget": "TextWidget",
+    classNames: "functional-unit",
+  },
+  measurementPeriodStartDate: {
+    "ui:widget": "DateWidget",
+  },
+  measurementPeriodEndDate: {
+    "ui:widget": "DateWidget",
+  },
+  baselineEmissionIntensity: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  targetEmissionIntensity: {
+    "ui:widget": "NumberWidget",
+    numberOfDecimalPlaces: 8,
+  },
+  postProjectEmissionIntensity: {
```

Comment:
deleted this and it's gone from db: 
![image](https://github.com/bcgov/cas-cif/assets/77306764/0256ddc8-e101-4f6d-8428-b5f266282dfa)